instrumentation/otel-webserver-module/build.gradle

```diff
@@ -65,10 +65,6 @@ project.ext {
 
     BOOST_INCLUDE = "${modDepDir}/boost/1.75.0/include"
     BOOST_LIB = "${modDepDir}/boost/1.75.0/lib/libboost_filesystem.a ${modDepDir}/boost/1.75.0/lib/libboost_system.a"
-    if (target_system == "ubuntu") {
```

Comment:
Ideally, we should not have this code. This is not there in main. Seems like some rebase error.

Reply:
Yeah there was a rebase error in the last push that's why I had to remove this. The rebase error was due to git